src/Form/Money.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace ZFBrasil\DoctrineMoneyModule\Form;
+
+use ZFBrasil\DoctrineMoneyModule\Exception\BadMethodCallException;
+use Zend\Filter\StringToUpper;
+use Zend\Form\Fieldset;
+use Zend\Form\Element\Number;
+use Money\Money as MoneyValueObject;
+use Money\Currency as CurrencyValueObject;
+use ZFBrasil\DoctrineMoneyModule\Form\Element\CurrencySelect;
+use ZFBrasil\DoctrineMoneyModule\Hydrator\MoneyHydrator;
+use Zend\InputFilter\InputFilterProviderInterface;
+use Zend\I18n\Filter\NumberFormat;
+use Zend\Validator\NotEmpty;
+
+/**
+ * Money form element that will make it very easy to work with money and currencies
+ *
+ * @author Fábio Carneiro <fahecs@gmail.com>
+ * @license MIT
+ */
+class Money extends Fieldset implements InputFilterProviderInterface
+{
+    /**
+     * @var CurrencyValueObject|null
+     */
+    protected $currency;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function init()
+    {
+        $this->setHydrator(new MoneyHydrator());
```

Comment:
It should be moved to the fieldset factory, in order to allow developers to use custom hydrators